sites/fast-website/src/public/header.ejs

```diff
@@ -8,8 +8,8 @@
             aria-label="Home"
             alt="FAST"
         >
-            <%= require("svg/icon-brand.svg") %> <%=
-            require("svg/icon-brand-fast.svg") %>
+            <img src="svg/icon-brand.svg">
+            <img src="svg/icon-brand-fast.svg">
```

Comment:
The html-bundler-webpack-plugin inlined SVG images with `type: "asset/source"` automatically.
Keep the HTML clean & clear using native `img` tag with source SVG file.